### PR TITLE
Updating gffread tests

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

No differences in logic really, just modified the tests to simplify the snapshot- helped me to debug use of this module in a workflow testing environment. 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/gffread/tests/main.nf.test [hunk 1]
```diff
@@ -25,10 +25,12 @@ nextflow_process {
 
         then {
             assertAll (
-            { assert process.success },
-            { assert snapshot(process.out).match() },
```
Comment (CarsonJM) [inline modules/nf-core/gffread/tests/main.nf.test:29]: Quick question; why not just use this assertion if it works? Shouldn't the `process.out.gffread_gff` be included in this assertion?

Comment (pinin4fjords) [reply modules/nf-core/gffread/tests/main.nf.test:29]: Using process.out (without a specific emit) causes the files to be duplicated in the snapshot, one for the numbered array, one for the named. By specifying which emits to test, we reduce that duplication. Not a biggie, just makes it easier to assess snapshot mismatch errors. 

These tests are configured such that the first produces only a gtf, the seond only the gff. So I'm only testing the snapshot for the one that is produced, leaving the not produced emit for the other assertion.

Comment (CarsonJM) [reply modules/nf-core/gffread/tests/main.nf.test:29]: Got it, thank you for the clarification! I definitely understand simplifying the snapshot, the duplication makes if much more difficult.

**Review by CarsonJM — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks @CarsonJM !

**Pull request merged**